src/style.css

```diff
@@ -0,0 +1,720 @@
+  gap: 4rem;
+}
+
+.nav-header h2 {
+  color: var(--color-black);
+  font-size: 1rem;
+}
+
+.nav-toggle {
+  font-size: 1.5rem;
+  color: #2980b9;
+  background: transparent;
+  border-color: transparent;
+  transition: var(--transition);
+  cursor: pointer;
+}
+
+.nav-toggle:hover {
+  color: var(--light-orange-clr);
+}
+
+.logo {
+  width: 25%;
+  padding: 0.6rem;
+  border: 1px solid indigo;
+  border-radius: 50%;
+  transition: var(--transition);
+}
+
+.logo:hover {
+  border: 4px solid #7a3d13;
+}
+
+.links a {
+  color: var(--background-clr);
+  font-size: 1.2rem;
+  text-transform: capitalize;
+  letter-spacing: var(--spacing);
+  display: block;
+  padding: 0.5rem 1.5rem;
+  transition: var(--transition);
+}
+
+.likes {
+  color: var(--color-black);
+  font-size: 1.2rem;
+  cursor: pointer;
+}
+
+.likes i {
+  color: orangered;
+}
+
+.links a:hover {
+  background: #d6dbdf;
+  color: var(--color-primary);
+  padding-left: 1.5rem;
+}
+
+.links {
+  height: 0;
+  overflow: hidden;
+  transition: var(--transition);
+}
+
+.show-links {
+  height: auto;
+}
+
+/* MOVIE SECTION */
+
+.movie-section {
+  margin: 12rem 1rem 4rem;
+}
+
+.box-container {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 2.5rem;
+}
+
+.box-container .box {
+  padding-bottom: 1rem;
+  background: var(--color-white);
+  border-radius: 0.8rem;
+  border: 5px solid rgb(235, 174, 6);
+  position: relative;
+  overflow: hidden;
+}
+
+.box .box-img {
+  margin: 1rem;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  overflow: hidden;
+}
+
+.box .box-img img {
+  border-radius: 0.6rem;
+  height: 20rem;
+  width: 100%;
+}
+
+.box .box-desc {
+  margin: 1rem;
+  display: flex;
+  align-items: center;
+  gap: 1.5rem;
+  justify-content: space-between;
+  padding: 0.5rem 1rem;
+}
+
+.box .box-desc .likes {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 1rem;
+}
+
+.btn-container {
+  display: flex;
+  flex-direction: column;
+  margin: 0 1rem;
+  gap: 1.9rem;
+}
+
+.btn-container .btn {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 0.6rem;
+  font-size: 1.2rem;
+  font-family: var(--font-family-primary);
+  border-radius: 0.4rem;
+  box-shadow:
+    rgba(26, 22, 24, 0.4) 5px 5px,
+    rgba(90, 46, 5, 0.3) 10px 10px,
+    rgba(37, 26, 33, 0.2) 15px 15px;
+  cursor: pointer;
+  transition: var(--transition);
+}
+
+.btn-container .btn:hover {
+  background: rgb(4, 32, 65);
+  border: var(--color-indigo);
+  color: var(--color-white);
+  transform: scale(0.9);
+}
+
+.alarm-form-reservations {
+  margin-top: -10px;
+  color: red;
+}
+
+/* POPUP SECTION */
+.popup-section {
+  background: black;
+}
+
+.popup-img,
+.reservation-img {
+  text-align: center;
+  margin-top: 1rem;
+  padding-top: 2rem;
+}
+
+.popup-section::-webkit-scrollbar {
+  width: 10px;
+}
+
+.popup-section::-webkit-scrollbar-thumb {
+  background: #f0f0f0;
+  border-radius: 2rem;
+}
+
```

Comment:
- [ ] From the project requirements, you expected to adhere to this given design in terms of the data displayed on the page by comparing your design and the given design your popup does not look like the given popup page design.

 comparing the differences... 
  - the given design has no nav for the popup page but I can still see your nav clearly for your popup page
  - I can clearly see your home page cards in the popup page

| Your Design |Given Design |
| :---:   | :---: | 
| ![Screenshot from 2023-01-05 23-32-57](https://user-images.githubusercontent.com/92303891/210900382-fadaaca8-dba5-4df5-9002-5345c16aa836.png) | ![image](https://user-images.githubusercontent.com/92303891/210899504-7815779e-8298-4669-8854-025a8d64ec17.png)|

I think is better if you enlarge the popup window and blur the background to differentiate the two pages ie. (the home page and the popup page)